### ISSUE TITLE
100% CPU load after import tensorflow as tf when using USB camera stream on Windows 10

### ISSUE THREAD

**Issue by Wunder0** [tensorflow/tensorflow#21809]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: b'v1.10.0-rc1-19-g656e7a2b34' 1.10.0
- **Python version**: 
- **Bazel version (if compiling from source)**: Python 3.5.5 :: Anaconda, Inc.
-**Mobile device: No Mobile device
- **CUDA/cuDNN version**: no GPU
- **GPU model and memory**: no GPU
- **Exact command to reproduce: python testCamera.py

### Describe the problem
I am using the pYuEye uEye API to get live images for my iDS industry USB 3.0 camera, see the simple attached code. When I run the code without importing tensorflow everything is fine. I get an image every 100 ms. But when I just add the line 'import tensorflow as tf' my CPU (i7 4770) has a 100% CPU load. I do not use any function from tensorflow.

Does anyone have an idea what could be the problem?

Cheers,
Wunder0
### Source code / logs
```
from Kameraobjekt import Kamera
import numpy as np
import tensorflow as tf


def startCam():
    EnableAutoparameter = True      
    EnableHardwareGain = False 
    AOI_x = 112        
    AOI_y = 84        
    AOI_width = 800     
    AOI_height = 600    
    Buffersize = 3    
    
    Cam.init()  
    Cam.SetParams(EnableAutoparameter, EnableHardwareGain)
    Cam.SetAOI(AOI_x, AOI_y, AOI_width, AOI_height)
    Cam.SetFPS(10.)  
    Cam.AllocMem(Buffersize)
    Cam.StartCaptureImage()
    Cam.setExposure()

Cam = Kamera()
startCam()
while True:
    Puffer = Cam.GetImage() 
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by Bumblebee269** [comment]
Just an observation, first you talk about every 100ms an image, after import 100% CPU Utilization. These are two different measures, did you got each 100ms an image after import tf?
Please note that ids recommends to disable idle states. If you do this, then usually the cpu utilization in Windows task manager is 100% even if your application does not do much. 
Maybe this helps.

**Comment by gomer-noah** [comment]
For posterity in case someone else encounters this issue (mine was importing torch instead of tensorflow, but I believe the issue is the same):

I was able to solve it by adjusting the OMP policy. Setting an environmental variable (OMP_NUM_THREADS = 2) greatly reduced CPU usage without slowing down my throughput. I went from using 100% of all 16 cores to using 100% of only two cores. Setting OMP_NUM_THREADS = 1 further reduced CPU load to using only 80% of a single core and didn't appear to affect throughput either. Still doing some testing though.
